draft-irtf-cfrg-vdaf.md

```diff
@@ -2713,6 +2720,11 @@ but this does allow uniqueness to be determined more efficiently.
 The algorithms below make use of the auxiliary function `decode_input_share()`
 defined in {{poplar1-auxiliary}}.
 
```

Comment:
Given the importance of this text, I think we should lift it to a subsection somewhere. We want to make sure it's not buried

Reply:
Done.